### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for document in "Create Authentication Files dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1384]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , tab to document and observe the name property using inspect.

**Actual Result:**
Name property is empty for the document..
Narrator is not announcing the value.value information for the document.

**Narrator behaviour:**
Tab

**NVDA behaviour:**
edit  read only
blank

**Expected Result:**
Name property should be simple and meaningful.
Name property for the document should be "Create Authentication Files".
Narrator should  announce the value.value for the document .

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![create authentication file-document](https://user-images.githubusercontent.com/22956130/35681775-76bdcf5e-0784-11e8-9ab8-cd4295bceac3.jpg)
![create authentication file-document-no name-nvda](https://user-images.githubusercontent.com/22956130/35681777-79f3ca02-0784-11e8-8254-6f46a5b9f22f.jpg)

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0.
Issue still exists
![1384- not fixed](https://user-images.githubusercontent.com/22956130/39797084-db235448-5377-11e8-8ada-fdc80600f9c8.jpg)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

The same with #1380 .

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150626-b898fb00-d309-11e8-89a3-0136e1451ee7.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749853
